packages/dd-trace/src/config.js

```diff
@@ -201,16 +201,22 @@ class Config {
       false
     )
 
-    let appsec = options.appsec || (options.experimental && options.experimental.appsec)
+    let appsec = options.appsec != null ? options.appsec : options.experimental && options.experimental.appsec
+
+    if (typeof appsec === 'boolean') {
+      appsec = {
+        enabled: appsec
+      }
+    } else if (appsec == null) {
+      appsec = {}
+    }
 
     const DD_APPSEC_ENABLED = coalesce(
-      appsec && (appsec === true || appsec.enabled === true), // TODO: remove when enabled by default
-      process.env.DD_APPSEC_ENABLED,
-      false
+      appsec.enabled && isTrue(appsec.enabled),
```

Comment:
The right side of this will not work properly with this operator. Is it even needed if `appsec.enabled` is a boolean anyways? `isTrue` is meant for strings.

Reply:
yeah ok i got confused here, i can just have
```suggestion
      appsec.enabled,
```
but it means any value passed will be evaluated a truthy/falsy